website/content/docs/developers/simulator-mode.md

```diff
@@ -39,8 +40,22 @@ simulator.deactivate("test");
 simulator.shutdown();
 ```
 
-To kill a topology, you could also simply terminate this process.
+To kill a topology, one could also terminate the process.
+
+The simulator mode will run in separate threads other than main thread. All those interfaces are 
+thread-safe. This means that one could invoke these interfaces in other threads and monitor the 
```

Comment:
again, what are "these interfaces"? I see methods on a Simulator class discussed above, not interfaces.